honeybee_energy/faceutil.py

```diff
@@ -0,0 +1,32 @@
+@property
+def face_to_idf(self):
```

Comment:
I am a little bit lost in terms of the relationship of the `Writer` class with the `ladybug_geometry Face3D` class and the overall `honeybee Face` class, which I imagine that the energy properties are extending.

Reply:
`Writer` has not direct relationship with `ladybug_geometry Face3D`.

Writer is used when one calls `Face.to` method to _write_ the face _to_ an another format.

Reply:
So are the vertices not being stored in a `ladybug_geometry Face3D`?  Are the vertices duplicated on the `Writer` and the `ladybug_geometry Face3D ` class?

Reply:
Vertices are stored in Face itself and are referenced from `Writer`.

Reply:
I changed how writer is structured so one need to send the `face` itself to writer and all the properties including vertices are accessed from face itself.

Reply:
This seems like the better way to do it even though I understand that it is not as elegant.